custom_connectors/none/two_checkout.rb

```diff
@@ -988,21 +988,17 @@
       end,
 
       webhook_notification: lambda do |_input, payload, _e_i_s, _e_o_s, _headers|
-        if payload['HASH'].present? && payload['MESSAGE_ID'].present?
-          payload&.each do |key, value|
-            if value.is_a?(Array)
-              payload[key] =
-                value&.map do |val|
-                  { 'value' => val } if val.is_a?(String)
-                end
-            else
-              value
-            end
```

Comment:
Why are we removing this logic?

Reply:
I am using the same logic. I just removed the condition, else it will miss few records.
```rb
if payload['HASH'].present? && payload['MESSAGE_ID'].present?
```

Reply:
Okay, I will add these guard clauses.